Makefile

```diff
@@ -6,7 +6,7 @@ PKG := github.com/Clever/configure
 PKGS := $(shell go list ./... | grep -v example)
 .PHONY: all test
 
-$(eval $(call golang-version-check,1.8))
+$(eval $(call golang-version-check,1.9))
```

Comment:
Can you leave this at 1.8 until we upgrade everything to 1.9? The golang.mk will prevent the versioning error from affecting you